### PR TITLE
[fix] make build error and cgroup memory. add Dockerfile for test

### PR DESCRIPTION
**Pull request by redtree0**
1. Edit a Makefile
2. Fix open() (add args mode)
3. Add a Dockerfile for test

### CODE REVIEW

File: src/diyc.c [hunk 1]
```diff
@@ -216,17 +216,21 @@ cgroup_setup(pid_t pid, unsigned int limit)
...
     fclose(fp);
 
     /* No swap */
     snprintf(cgroup_file, PATH_MAX, "%s/memory.memsw.limit_in_bytes", cgroup_dir);
-    fp = fopen(cgroup_file, "w");
-    if (NULL == fp) die("Could not set swap limit");
-    fprintf(fp, "0\n");
-    fclose(fp);
+
+    if(access(cgroup_file, R_OK | W_OK) == 0){
```
Comment (w-vi) [inline src/diyc.c]:
```suggestion
    if(access(cgroup_file, R_OK | W_OK) == 0) {
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM debian
+
+MAINTAINER redtree0 redtree941@naver.com
```
Comment (w-vi) [inline Dockerfile]:
```suggestion

```

Comment (w-vi) [reply Dockerfile]: As a maintainer of the repo I will take care of the Dockerfile too.

**Review by w-vi — CHANGES_REQUESTED** [review]
Thanks a lot for the contribution, I appreciate the effort to improve it. I have added few suggestions and please can you squash the code changing commits into one? The `.gitignore` and `Dockerfile` should be separate commits so the history is clear. Thanks a lot.

**Comment by redtree0** [comment]
@w-vi Thanks for your advice. I think that I tried your comment. Please let me know if you need more. Thank you. 😆

**Review by w-vi — APPROVED** [review]

**Comment by w-vi** [comment]
Thanks a lot!

**Pull request merged**